src/transform-sdk/rust/sr/src/lib.rs

```diff
@@ -80,10 +80,54 @@ impl Default for SchemaRegistryClient {
     }
 }
 
+#[deprecated]
```

Comment:
Can we add a note on what to replace it with?

https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute

Reply:
done